src/components/BatchClassComponents/DoubtClassExerciseComponent.js

```diff
@@ -0,0 +1,349 @@
+import React, { useEffect, useState } from "react";
+import useMediaQuery from "@mui/material/useMediaQuery";
+import { Typography, Card, CardContent, Chip } from "@mui/material";
+import { Button } from "@mui/material";
+import { Box } from "@mui/system";
+import useStyles from "./styles";
+import { breakpoints } from "../../theme/constant";
+import ArrowForwardIosIcon from "@mui/icons-material/ArrowForwardIos";
+
+import SwipeableDrawer from "@mui/material/SwipeableDrawer";
+
+import { lang, TimeLeft } from "../../constant";
+import { format } from "../../common/date";
+import AlertDialog from "./AlertDialog";
+import DropOut from "./DropOut";
+
+export const MoreDetails = (props) => {
+  const { open, setOpen, isEnrolled, setIsEnrolled } = props;
+
+  const isActive = useMediaQuery("(max-width:" + breakpoints.values.sm + "px)");
```

Comment:
Not used for this component

Reply:
amansharmma uses this everywhere and when he creates new component he copies the snippet of code from elsewhere don't know why??
back in days I explained him use snippets from extension rather than using the copy paste from somewhere else.
now it wont happen again.
regards
@jschanker

Reply:
@jschanker @komalahire And @Poonam-Singh-Bagh are working on removing such things like removing unused vars

Reply:
not sure about test failed thing